File: _examples/lua/tuples.lua
-- Lua 使用 table 模拟元组
local person = {25, "Alice", 1.68}

-- 访问元素
local age = person[1]
local name = person[2]

-- 多重赋值（类似解包）
local a, b, c = 25, "Alice", 1.68

-- 函数返回多值
local function getPerson()
    return 25, "Alice", 1.68
end
local age, name, height = getPerson()

File: _examples/go/tuples.go
// Go 没有元组类型
// 使用多返回值模拟
func getPerson() (int, string, float64) {
    return 25, "Alice", 1.68
}

age, name, height := getPerson()

// 或使用结构体
type Person struct {
    Age    int
    Name   string
    Height float64
}